### ISSUE TITLE
How to add more than one AC

### ISSUE THREAD

**Issue by mmoti6** [ghosty2004/homebridge-connectlife-ac#6]
**Describe Your Problem:**
i have 5 AC 
How I can connect all of them?

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by ghosty2004** [comment]
Hi. I fixed in latest release, can you test it and confirm if it works?

Thanks.

**Comment by mmoti6** [comment]
I add 4 AC and Now the plugin cannot start 
I will reinstall the plugin and update here

**Comment by ghosty2004** [comment]
> Now the plugin cannot start 

Can you show the logs?

**Comment by mmoti6** [comment]
![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/7649abd9-5ad1-4f4e-a270-6d08fb5f4c7b)
![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/2d552a39-4783-4781-b1e9-e0ca39c03481)

**Comment by ghosty2004** [comment]
> ![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/7649abd9-5ad1-4f4e-a270-6d08fb5f4c7b)
> ![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/2d552a39-4783-4781-b1e9-e0ca39c03481)

Have you tried to reinstall the plugin (how I noticed everyone in the release information?) If no, then reinstall it (uninstall and install again)

**Comment by mmoti6** [comment]
Yes and still not work ![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/93e33cd1-5d4a-498c-b4ae-2f801e12c904)

And the ac in homekit not responding

**Comment by ghosty2004** [comment]
> Yes and still not work ![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/93e33cd1-5d4a-498c-b4ae-2f801e12c904)
> 
> And the ac in homekit not responding 

Can you confirm the following: if you add only one AC the plugin still crash? Thanks

**Comment by mmoti6** [comment]
I deleted 3 AC and only 1 remaining 

In The homekit work but this is what it looks like 

![image](https://github.com/ghosty2004/homebridge-connectlife-ac/assets/135166891/3c438498-3dc0-42a4-b10c-2a75f656e9f0)
